Core/src/org/sleuthkit/autopsy/datamodel/ImageNode.java

```diff
@@ -200,6 +204,22 @@ public String getItemType() {
         return getClass().getName();
     }
 
+    private Boolean checkSchemaVersion() {
+        try {
+            int creationMajorVersion = Case.getCurrentCaseThrows().getSleuthkitCase().getDBCreationSchemaVersion().getMajor();
+            int creationMinorVersion = Case.getCurrentCaseThrows().getSleuthkitCase().getDBCreationSchemaVersion().getMinor();
+        
```

Comment:
The SleuthkitCase method names on line 209 and 210 need to be getDBSchemaCreationVersion. Also, it would be a bit tidier to call getDBSchemaCreationVersion just once, in my opinion.